sdk/mobile-center-analytics/src/main/java/com/microsoft/azure/mobile/analytics/Analytics.java

```diff
@@ -121,6 +128,15 @@ public static void setEnabled(boolean enabled) {
     }
 
     /**
+     * Sets an analytics listener.
+     *
+     * @param listener The custom crashes listener.
+     */
+    public static void setListener(AnalyticsListener listener) {
+        getInstance().mAnalyticsListener = listener;
```

Comment:
Can you try changing to protected and use android bindings config to change it back to public?